cosmic_ray/operators/binary_operator_replacement.py

```diff
@@ -4,28 +4,25 @@
 from .operator import Operator
 from ..util import build_mutations
 
-
 OPERATORS = (ast.Add, ast.Sub, ast.Mult, ast.Div, ast.FloorDiv, ast.Mod,
              ast.Pow, ast.LShift, ast.RShift, ast.BitOr, ast.BitXor,
              ast.BitAnd)
 
 # todo: this often leads to unsupported syntax
 if sys.version_info >= (3, 5):
-    OPERATORS = OPERATORS + (ast.MatMult, )
+    OPERATORS = OPERATORS + (ast.MatMult,)
 
 
-def _to_ops(from_op):
-    """
-        The sequence of operators which `from_op` could be mutated to.
-    """
+def _to_ops(from_op):  # pylint: disable=unused-argument
+    """The sequence of operators which `from_op` could be mutated to."""
     for to_op in OPERATORS:
         yield to_op
 
 
 class MutateBinaryOperator(Operator):
     """An operator that modifies binary operators."""
 
-    def visit_BinOp(self, node):
+    def visit_BinOp(self, node):  # pylint: disable=invalid-name
```

Comment:
Was this also flagged as an invalid name?

Reply:
BinOp is not pep-8 compliant, at best could be 'bin_op'